src/components/PopoverWithoutOverlay/index.tsx

```diff
@@ -49,12 +49,14 @@ function PopoverWithoutOverlay(
         let removeOnClose: () => void;
         if (isVisible) {
             onModalShow();
+
+            const noop = () => {};
             onOpen?.({
                 ref: withoutOverlayRef,
-                close: onClose,
+                close: onClose ?? noop,
                 anchorRef,
             });
-            removeOnClose = setCloseModal(onClose);
+            removeOnClose = setCloseModal(onClose ?? (() => {}));
```

Comment:
noop could be imported from lodash or declared at the begining of file. In both cases you can use it in the line 59